src/shared/ThemeContext/ThemeContext.tsx

```diff
@@ -29,8 +29,15 @@ interface ThemeContextProviderProps {
 export const ThemeContextProvider: FC<ThemeContextProviderProps> = ({
   children,
 }) => {
-  const currentTheme = (localStorage.getItem('theme') as Theme) || Theme.LIGHT
-  const [theme, setTheme] = useState<Theme>(currentTheme)
+  const prefersDark = window.matchMedia('(prefers-color-scheme: dark)').matches
```

Comment:
This is the only logic changes

Reply:
I did consider that approach, though I ended up on preferring to have all the theme logic in one place